popover-api/popover-hint/index.css

```diff
@@ -0,0 +1,148 @@
+/* General styling */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: sans-serif;
+  height: 100%;
+}
+
+body {
+  height: inherit;
+  background: #ccc;
+  margin: 0;
+}
+
+#wrapper {
+  height: inherit;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* Button bar styling */
+
+#button-bar {
+  height: 48px;
+  width: 30%;
+  min-width: 300px;
+  padding: 8px;
+  border-radius: 6px;
+  background: #eee;
+  box-shadow: 1px 1px 3px #999;
+  display: flex;
+  justify-content: space-around;
+  gap: 10px;
+}
+
+/* Button styling */
+
+#button-bar button {
+  border: 1px solid #999;
+  border-radius: 4px;
+  background: #eee;
+  flex: 1;
+}
+
+#button-bar button:hover,
+#button-bar button:focus {
+  background: #ccc;
+}
+
+#button-bar button:active {
+  box-shadow: inset 1px 1px 2px #333, inset -1px -1px 2px #eee;
+}
+
+/* Anchoring and styling auto popovers */
+
+#menu-1 {
+  anchor-name: --menu-anchor-1, --tooltip-anchor-1;
+}
+
+#menu-2 {
+  anchor-name: --menu-anchor-2, --tooltip-anchor-2;
+}
+
+#menu-3 {
+  anchor-name: --menu-anchor-3, --tooltip-anchor-3;
+}
+
+[popover="auto"] {
+  inset: unset;
+  position: absolute;
+  bottom: calc(anchor(top) + 20px);
+  justify-self: anchor-center;
+  margin: 0;
+
+  width: 200px;
+  text-align: center;
+  padding: 8px;
+  border-radius: 6px;
+  background: #eee;
+  box-shadow: 1px 1px 3px #999;
+  border: none;
+}
+
+[popover="auto"] button {
+  border: 1px solid #999;
+  border-radius: 4px;
+  background: #eee;
+  width: 100%;
+  line-height: 1.5;
+}
+
+[popover="auto"] button:first-of-type {
+  margin-bottom: 8px;
+}
+
+[popover="auto"] button:hover,
+[popover="auto"] button:focus {
+  background: #ccc;
+}
+
+[popover="auto"] button:active {
+  box-shadow: inset 1px 1px 2px #333, inset -1px -1px 2px #eee;
+}
+
+#submenu-1 {
+  position-anchor: --menu-anchor-1;
```

Comment:
You shouldn't need to add `position-anchor` or `anchor-name` to anything in these demos, if you just add `showPopover({source: invoker})` to the JS bit. Declarative `popovertarget` activation automatically creates an implicit anchor reference, as does using `source` in `showPopover()`.

Reply:
@mfreed7 Nice! That works like a charm. Updated.

QQ: We don't actually have the `show` option documented on MDN. I'm going to add it in the PR associated with this demo, but I should also add a compat data entry for it at some point. When was `show` first supported?

Reply:
Great! Yeah, I think that's an outstanding item: https://github.com/mdn/mdn/issues/598. It's been shipped in Chromium since M133.